smart-contracts/osmosis/contracts/cl-vault/tests/test-tube/autocompound.rs

```diff
@@ -266,98 +268,113 @@ fn test_autocompound_with_rewards_swap_non_vault_funds() {
         &admin,
     )
     .unwrap();
-
-    // Assert contract balance about the just airdropped non vault token funds
-    let balances_rewards =
+    let initial_rewards_balance =
         get_balance_amount(&app, contract_address.to_string(), DENOM_REWARD.to_string());
     assert_eq!(
         DENOM_REWARD_AMOUNT.to_string(),
-        balances_rewards.to_string(),
+        initial_rewards_balance.to_string(),
     );
 
     // SWAP NON VAULT ASSETS BEFORE AUTOCOMPOUND ASSETS
 
-    // Swap non vault funds to vault funds
-    // 50000000000ustride to 49500000000uatom as spot price 1.0 less swap_fees
-    // 50000000000ustride to 49500000000uosmo as spot price 1.0 less swap_fees
-    let expected_amount_rewards = 49500000000u128;
+    // Calculate the total rewards swap amount using balanced_fee configs (0.01 is coming from osmosis as flat fee)
```

Comment:
What is balanced_fee configs? And why is it important that 0.01 is a "flat" fee?
to me the code after is clearer than the comment which leads to questions that are not relevant for this test case. I'd prefer to remove that

Reply:
Is a flat fee on test_tube https://github.com/osmosis-labs/test-tube/blob/main/packages/osmosis-test-tube/src/module/gamm.rs#L59 as the Gamm module from it is hardcoding 0.01% swap fee and we cannot change that.